### DISCUSSION TITLE
Ignore DB sync

### DISCUSSION THREAD

**Discussion by Kinfe123** [withastro/roadmap#864] [Proposal]
### Body

What about there is a cli prefix or a kind of preset that can ignore making HMR on the db while `astro dev` avoiding dev runtime overhead .. i am not quite sure but somehow it affect the dev perfomance for non db stuff

**Comment by Kinfe123** [comment]
does this work with package.json or something like that

**Comment by florian-lefebvre** [comment]
That sounds like a bug tbh! Would you mind opening an issue on the main rapidement with a minimal reproduction? Thank you!

**Comment by florian-lefebvre** [comment]
Astro DB has been deprecated
